lacommunaute/forum/management/commands/populate.py

```diff
@@ -14,7 +14,10 @@ class Command(BaseCommand):
     help = "hydratation d'un site de validation"
 
     def handle(self, *args, **options):
-        UserFactory(username="communaute", password=make_password("password"))
+        user = UserFactory(username="communaute", password=make_password("password"))
+        user.is_superuser = True
+        user.is_staff = True
+        user.save()
```

Comment:
Évite de faire `INSERT ...` puis `UPDATE ...`
```suggestion
        user = UserFactory(username="communaute", password=make_password("password"), is_superuser=True, is_staff=True)
```

Aussi, si tu veux éviter le `make_password` : https://factoryboy.readthedocs.io/en/stable/reference.html#transformer

Reply:
cool le transformer, merci